checkov/terraform/checks/resource/aws/RDSInstanceDeletionProtection.py

```diff
@@ -0,0 +1,18 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class RDSInstanceDeletionProtection(BaseResourceValueCheck):
+
+    def __init__(self):
+        name = "Ensure that AWS database instance have deletion protection enabled"
```

Comment:
```suggestion
        name = "Ensure that AWS database instances have deletion protection enabled"
```
or
```suggestion
        name = "Ensure that AWS database instance has deletion protection enabled"
```

Reply:
I have pushed the said changes as  name = "Ensure that AWS database instances have deletion protection enabled" . Let me know if more information required ! :)